cmd/start.go

```diff
@@ -16,6 +16,7 @@ const (
 	bearerTokenFlagName    = "bearer-token"
 	corsFlagName           = "cors-origin"
 	evaluatorFlagName      = "evaluator"
+	grpcCertPath           = "grpc-cert-path"
```

Comment:
Eventually we might want this to be per sync... is that right?

Reply:
Yes, this makes sense. I will update the name.